osgi-dependencies/proxy/src/main/java/com/cognifide/aet/proxy/ProxyServerProvider.java

```diff
@@ -44,19 +44,20 @@
           unbind = "unbindProxyManager")
   private final Map<String, ProxyManager> collectorManagers = Maps.newConcurrentMap();
 
-  public ProxyServerWrapper createProxy(String useProxy, int port) throws ProxyException {
-    String proxyType = useProxy;
-    if ("true".equals(useProxy)) {
+  public ProxyServerWrapper createProxy(String usedProxy) throws ProxyException {
+    String proxyType = usedProxy;
+    if ("true".equals(usedProxy)) {
```

Comment:
I have created new issue for that: https://github.com/Cognifide/aet/issues/56

Could you please approve / update the description when you have time?